src/extension/features/accounts/reconcile-balance/index.ts

```diff
@@ -12,29 +12,30 @@ export class ReconcileBalance extends Feature {
   }
 
   shouldInvoke() {
-    let { selectedAccountId } = controllerLookup('accounts');
-    return selectedAccountId && isCurrentRouteAccountsPage();
+    return isCurrentRouteAccountsPage();
   }
 
   invoke() {
     // Get the current account id and calculate the current reconciled balance
-    let { selectedAccountId } = controllerLookup('accounts');
-    let reconciledBalance = formatCurrency(this._calculateReconciledBalance(selectedAccountId));
+    let accountsController = controllerLookup<YNABAccountsController>('accounts');
+    if (accountsController !== undefined) {
```

Comment:
instead of wrapping this whole block in an if, can we do something like this?
```suggestion
    let accountsController = controllerLookup<YNABAccountsController>('accounts');
    if (accountsController === undefined) return;
```

Reply:
Fixed on https://github.com/toolkit-for-ynab/toolkit-for-ynab/commit/9ec32cb0c39de07945e1c4cf5b2246b250c6823d.